Sources/JobsPostgres/Migrations/CreateJobs.swift

```diff
@@ -24,11 +24,18 @@ struct CreateJobs: PostgresMigration {
                 id uuid PRIMARY KEY,
                 job bytea,
                 status smallint,
-                lastModified TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
+                last_modified TIMESTAMPTZ DEFAULT NOW()
             )     
             """,
             logger: logger
         )
+        try await connection.query(
```

Comment:
Is this to improve the performance of the queries called from `onInit`?

Reply:
Yes, it is.